src/Abp.ZeroCore/Authorization/Users/UserTokenExpirationWorker.cs

```diff
@@ -35,17 +34,15 @@ public class UserTokenExpirationWorker<TTenant, TUser> : PeriodicBackgroundWorke
             _unitOfWorkManager = unitOfWorkManager;
             _tenantRepository = tenantRepository;
 
-            Timer.Period = GetTimerPeriod();
```

Comment:
The abstraction actually looks neater.

Reply:
This looks neater but as you mentioned the configuration should be only used once. i think it better to read the period value directly in constructor